pages/_tutorials/howto/howto_add_markup.md

```diff
@@ -50,25 +54,89 @@ bioschemas:
+* @id --so it is clear what the identifier of the thing you are describing is
+* dct:conformsTo --so it is clear what Bioschemas profile version you adhere to
+
+For instance, if you are describing a `DataCatalog`, you should include the standards properties at the beginning of the markup as follows:
+
+```
+{
+  "@context": "https://schema.org/",
+  "@type": "DataCatalog",
+  "@id" : "https://www.guidetopharmacology.org",
+  "http://purl.org/dc/terms/conformsTo": {
+      "@type": "CreativeWork",
+      "@id": "https://bioschemas.org/profiles/DataCatalog/0.3-RELEASE-2019_07_01/"
+  },
+  ...
+}
+```
+
+### 3.3. How add the markup
+
+Here we have two possibilities: manually or automatically adding the markup.
+
+Depending on how the data is rendered on your website, there might be a production pipeline behind your web pages. If this is the case, web developers will decide the best way to programmatically add the Bioschemas markup. This is commonly the case for registries/repositories or data/knowledge bases. For instance, the [Ensembl Genome Browser](https://www.ensembl.org) provides information on genomes and has integrated the Bioschemas markup to their production pipeline. Thus, whenever you visit an Ensembl gene page you get not only the HTML but also the JSON-LD embedded within it.
+
+For websites without a production pipeline behind, those where you manually edit the HTML yourself, you will have to add the markup by hand. For instance, if you organize a half-day workshop with less than 10 accepted papers, you could add the corresponding markup by hand. This is the case for the [Research Objects Management for Linked Open Science Workshop](https://zbmed.github.io/damalos) which includes bibliographic markup for the [submissions on 2020](https://zbmed.github.io/damalos/docs/2020.html). The workshop web page is based on GitHub pages and the markup for the publications was manually added at the end of the corresponding [MarkDown document](https://github.com/zbmed/damalos/blob/master/docs/2020.md).
+
+### 3.4. Where to add the markup
+
+You can put the JSON-LD corresponding to your markup anywhere within the HTML where a ```<script>``` element is allowed. You can choose between having multiple ```<script type="application/ld+json">``` elements or only one. If you really need to you can even mix JSON-LD and RDFa, although we don't recommend this as many tools will not be able to extract all your markup.
+
+If you choose to have multiple ```<script type="application/ld+json">``` elements, most likely you want them close to the corresponding HTML. For instance, if you visit the gene page [BRCA2](https://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139618;r=13:32315086-32400268) at Ensembl, you will observe the following gene summary.
+
+{% include image.html file="/tutorials/images/brca_gene_summary.png" caption="Figure 1: BRCA2 gene summary" alt="BRCA2 gene summary" %}
+
+<br/>
```

Comment:
Maybe I add extra margin under each caption/picture so you don't have to use `<br>` ? :)

Reply:
<br/> removed